cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,21 @@
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
+    cy.visit('https://react-mobx.realworld.io/#/register');
+
+    cy.get('[placeholder=Username]')
+      .type(user.username);
+
+    cy.get('[placeholder=Email]')
+      .type(user.email);
+
+    cy.get('[placeholder=Password]')
+      .type(user.password);
+
+    cy.get('.btn').click();
```

Comment:
Selecting elements by their CSS class can lead to fragile tests because classes often change as a result of style changes. It's better to use a more specific selector, or add a specific attribute or ID to the element you want to select.